src/browser_launchers/puppeteer_launcher.js

```diff
@@ -33,8 +33,8 @@ const LAUNCH_PUPPETEER_DEFAULT_VIEWPORT = {
  * ```
  *
  * @typedef PuppeteerLaunchContext
- * @property {LaunchOptions} [launchOptions]
- *  `puppeteer.launch` [options](https://pptr.dev/#?product=Puppeteer&version=v5.5.0&show=api-puppeteerlaunchoptions)
+ * @property {Parameters<PuppeteerNode['launch']>[0]} [launchOptions]
```

Comment:
same here, i kinda expect this won't work with the jsdoc extraction properly, but lets merge and see